### PR TITLE
Ubuntu support:

### PR DESCRIPTION
**Pull request by mveytsman**
- Get distro & release for server
- Differentiate between gemfiles and dpkgs

### CODE REVIEW

File: agent/agent.go [hunk 1]
```diff
@@ -28,13 +28,21 @@ func NewAgent(conf *Conf, clients ...Client) *Agent {
...
+
 	return agent
 }
 
 // instantiate structs, fs hook
 func (agent *Agent) StartWatching() {
 	for _, f := range agent.conf.Files {
-		agent.files = append(agent.files, NewWatchedFileWithHook(f.Path, agent.OnFileChange))
+		q := NewWatchedFileWithHook(f.Path, agent.OnFileChange)
+		log.Info("BLAH", q)
```
Comment (phillmv) [inline agent/agent.go]: blah?

File: agent/models/server.go [hunk 2]
```diff
@@ -1,17 +1,58 @@
...
 
 import (
+	"io/ioutil"
 	"net"
 	"os"
 	"os/exec"
+	"strings"
 )
 
+var UBUNTU_RELEASES = map[string]string{
```
Comment (phillmv) [inline agent/models/server.go]: move to server, if you don't need the name

**Comment by phillmv** [comment]
seems aight

**Comment by mveytsman** [comment]
@phillmv apporves

**Pull request merged**